app/src/main/java/com/example/keniphhomemaintenance/dwellings/DwellingItem.kt

```diff
@@ -0,0 +1,82 @@
+package com.example.keniphhomemaintenance.dwellings
+
+import androidx.compose.foundation.layout.Column
+// import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.CornerSize
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Button
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme.typography
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import com.example.keniphhomemaintenance.ui.Screen
+
+val dwellingsViewModel = DwellingsViewModel()
```

Comment:
Data should be passed in to the DwellingItem via parameters, and probably don't need the whole view model, just the data that is relevant to the DwellingItem. I'm guessing we'll want the viewmodel to live at the "screen" level (DwellingsScreenView), which creates the DwellingItems by only passing the DwellingItems the information they need.